diffkemp/simpll/simpll.py

```diff
@@ -19,10 +23,14 @@ def add_suffix(file, suffix):
 
 
 def simplify_modules_diff(first, second, fun_first, fun_second, var,
-                          suffix=None, control_flow_only=False,
-                          print_asm_diffs=False, verbose=False):
+                          suffix=None, equal_funs=None,
+                          control_flow_only=False, print_asm_diffs=False,
+                          verbose=False):
```

Comment:
I think that this is a good time to rename this function, since it no longer has much to do with simplifying, it performs semantic comparison running SimpLL. Perhaps the best name would be `run_simpll`.